Dockerfile

```diff
@@ -14,13 +14,18 @@ RUN cat /etc/os-release \
     && systemctl enable gcp-routes gcp-hostname \
     && cp -irvf bootstrap / \
     && cp -irvf manifests / \
+    && cp -ivf okd-copr.repo /etc/yum.repos.d \
     && rpm-ostree install \
         NetworkManager-ovs \
         open-vm-tools \
         qemu-guest-agent \
-    && rpm -Uvh /tmp/rpms/* \
+        cri-o \
+        cri-tools \
+        /tmp/rpms/openshift-clients-[0-9]*.rpm \
```

Comment:
Is this due to `openshift-clients-redistributable.rpm`? Perhaps we should not include it in `okd-rpms`?

Reply:
yes. I was thinking we could instead drop `rpms` from this build now, and get the rpms from the artifacts directly.
In the longer run, I'd like to restructure the `artifacts` image to contain a proper yum repo (not a weirdly merged one). The `okd-rpms` would for now be only AI, until we make an OKD variant of the `artifacts` image that also contain's cri-o